hydrax/simulation/deterministic.py

```diff
@@ -186,6 +251,13 @@ def run_interactive(  # noqa: PLR0912, PLR0915
                 mujoco.mj_step(mj_model, mj_data)
                 viewer.sync()
 
+                # Capture frame if recording
+                if record_video:
```

Comment:
it's slightly redundant, but maybe we should do
```suggestion
                if record_video and process is not None:
```
to telegraph intention a bit more explicitly (and to match the control flow logic at the end of the script)